lib/components/artist/shows/variable_size_shows_list.js

```diff
@@ -3,19 +3,29 @@
 
 import Relay from 'react-relay'
 import React from 'react'
-import type { LayoutEvent } from '../../system/events'
-import { Dimensions, View, StyleSheet } from 'react-native'
+import { View, StyleSheet } from 'react-native'
 
 import Show from './show'
 
-class LargeList extends React.Component {
+class ShowsList extends React.Component {
   state: {
     width: number,
-    height: number
+    height: number,
   }
 
   constructor(props) {
     super(props)
+
+    // The 'showSize' must exist and should be a value this component accommodates
+    if (__DEV__) {
+      const possibleShowSizes = [ 'medium', 'large' ]
```

Comment:
Might want to define an enum by using literal types https://flowtype.org/docs/builtins.html#literal-types (scroll a bit down).